src/org/labkey/test/components/DomainDesignerPage.java

```diff
@@ -21,9 +21,19 @@ public static DomainDesignerPage beginAt(WebDriverWrapper driver, String contain
         return new DomainDesignerPage(driver.getDriver());
     }
 
-    public DomainDesignerPage clickSaveChanges()
+    public DomainDesignerPage clickSave()
     {
-        elementCache().saveChangesBtn.click();
+        elementCache().saveButton.click();
+        return this;
+    }
+    public WebElement saveButton()
+    {
+        return elementCache().saveButton;
+    }
+
+    public DomainDesignerPage clickSaveAndFinish()
+    {
+        elementCache().saveAndFinishButton.click();
```

Comment:
`Save and Finish` usually navigates away. Waiting for a page load would be less prone to weird timing errors:
```suggestion
        clickAndWait(elementCache().saveAndFinishButton);
```

Reply:
done